app/models/appointment_slot.rb

```diff
@@ -51,7 +68,7 @@ def self.for_date(date)
 
     # Generate slots based on preset if no special slot have been specified for that date
     AppointmentSlotPreset
-      .where(day: date.wday)
+      .where(day: date.wday + 1)
```

Comment:
Will this change affect any existing feature?

Reply:
Ah good catch ! I was halfway on fixing a bug : )

It's a bug that got revealed by the tests (`wdate` is 0-indexed as opposed to our code which is 1-indexed). So Sundays are missed (which has no consequence, but still good to fix). Pushing the fix now